Directory.Build.targets

```
@@ -8,14 +8,14 @@
     necessary as the output of this leg is used in other CI source build legs. Those could be
     targeting NetCurrent or NetPrevious hence we must produce both.
   -->
-  <PropertyGroup Condition=" '$(DotNetBuildFromSource)' == 'true' AND '$(DotNetBuildFromSourceFlavor)' != 'Product' ">
+  <PropertyGroup Condition=" '$(DotNetBuildSourceOnly)' == 'true' and '$(DotNetBuild)' != 'true' ">
```

```suggestion
  <PropertyGroup Condition=" '$(DotNetBuildSourceOnly)' == 'true' and '$(DotNetBuildOrchestrator)' != 'true' ">
```

This pattern is about repo level source build vs. building as the whole product. Should be the above.

---

Repo level source build doesn't set DotNetBuild though, or does it?